### PR TITLE
chore: remove unused code

### PR DESCRIPTION
**Pull request by walldiss**
Fine collection of skeletons

### CODE REVIEW

File: share/shwap/sample.go [hunk 1]
```diff
@@ -14,9 +14,6 @@ import (
 	"github.com/celestiaorg/celestia-node/share/shwap/pb"
 )
 
-// SampleName is the name identifier for the sample container.
-const SampleName = "sample_v0"
-
```
Comment (Wondertan) [inline share/shwap/sample.go:19]: Keep the names, for future shrex support

File: share/empty.go [hunk 2]
```diff
@@ -28,18 +28,11 @@ func EmptyEDS() *rsmt2d.ExtendedDataSquare {
 	return emptyBlockEDS
 }
 
-// EmptyBlockShares returns the shares of the empty block.
-func EmptyBlockShares() []libshare.Share {
-	initEmpty()
-	return emptyBlockShares
-}
-
```
Comment (Wondertan) [inline share/empty.go:36]: I would keep this one. You never know

File: header/serde.go [hunk 3]
```diff
@@ -64,37 +64,6 @@ func UnmarshalExtendedHeader(data []byte) (*ExtendedHeader, error) {
...
-	pb.ValidatorSet = valSet
-	dah, err := eh.DAH.ToProto()
-	if err != nil {
-		return nil, err
-	}
-	pb.Dah = dah
-	return pb, nil
-}
-
-func ProtoToExtendedHeader(pb *header_pb.ExtendedHeader) (*ExtendedHeader, error) {
```
Comment (Wondertan) [inline header/serde.go:85]: Wait, how do we serialize then?

File: share/empty.go (hunk 2 above)
Comment (walldiss) [reply share/empty.go:36]: Shares can be obtained from chained call EmptyEDS.FlattenedODS(), so there is not much value in additional global stored shares

File: header/serde.go (hunk 3 above)
Comment (walldiss) [reply header/serde.go:85]: We serialise by MarshalExtendedHeader/UnmarshalExtendedHeader. Those are converter fucntions, that are not really needed

File: share/empty.go (hunk 2 above)
Comment (Wondertan) [reply share/empty.go:36]: But its ods, not eds

Comment (walldiss) [reply share/empty.go:36]: The idea is similar, there is `Flattened()` for all eds shares

**Review by Wondertan — APPROVED** [review]

**Review by cristaloleg — APPROVED** [review]

**Pull request merged**